src/app/models/searchResults.ts

```diff
@@ -1,5 +1,6 @@
 export interface SearchResults {
```

Comment:
are we using this interface?

Reply:
yes I used it in order to get search results.

Reply:
I lied I didnt use it